examples/experimental/u7s.yaml

```diff
@@ -0,0 +1,145 @@
+# Deploy kubernetes via usernetes.
+# $ limactl start ./u7s.yaml
+# $ limactl shell u7s kubectl
+
+# It can be accessed from the host by exporting the kubeconfig file;
+# the ports are already forwarded automatically by lima:
+#
+# $ export KUBECONFIG=$(limactl list u7s --format 'unix://{{.Dir}}/copied-from-guest/kubeconfig.yaml')
+# $ kubectl get no
+# NAME           STATUS   ROLES           AGE   VERSION
+# u7s-lima-u7s   Ready    control-plane   33s   v1.28.0
+
+# This template requires Lima v0.8.0 or later
+images:
+# Try to use release-yyyyMMdd image if available. Note that release-yyyyMMdd will be removed after several months.
+- location: "https://cloud-images.ubuntu.com/releases/22.04/release-20231010/ubuntu-22.04-server-cloudimg-amd64.img"
+  arch: "x86_64"
+  digest: "sha256:5bed3f233c2422187e86089deea51bb8469dc2a26e96814ca41ff8f14dc80308"
+- location: "https://cloud-images.ubuntu.com/releases/22.04/release-20231010/ubuntu-22.04-server-cloudimg-arm64.img"
+  arch: "aarch64"
+  digest: "sha256:5167c1b13cb33274955e36332ecb7b14f02b71fd19a37a9c1a3a0f8a805ab8e5"
+# Fallback to the latest release image.
+# Hint: run `limactl prune` to invalidate the cache
+- location: "https://cloud-images.ubuntu.com/releases/22.04/release/ubuntu-22.04-server-cloudimg-amd64.img"
+  arch: "x86_64"
+- location: "https://cloud-images.ubuntu.com/releases/22.04/release/ubuntu-22.04-server-cloudimg-arm64.img"
+  arch: "aarch64"
+
+# Mounts are disabled in this template, but can be enabled optionally.
+mounts: []
+# containerd is managed by Docker, not by Lima, so the values are set to false here.
+containerd:
+  system: false
+  user: false
+provision:
+# See [KEP-2033](https://features.k8s.io/2033) Kubelet-in-UserNS (aka Rootless mode)
+- mode: system
+  script: |
+    #!/bin/bash
+    set -eux -o pipefail
+    test -d /var/lib/usernetes && exit 0
+    branch=gen2-v20230919.0
```

Comment:
tag, not branch

Reply:
I suppose it is the same parameter, especially if we use git instead.

Reply:
Yes, but the variable is just a misnomer